app/code/core/Mage/Adminhtml/Block/Cache.php

```diff
@@ -30,7 +30,7 @@ public function __construct()
         parent::__construct();
         $this->_removeButton('add');
         $this->_addButton('flush_magento', [
-            'label'     => Mage::helper('core')->__('Flush OpenMage Cache'),
+            'label'     => Mage::helper('core')->__('Flush & Apply Updates'),
```

Comment:
I think we kinda decided not to change the labels in the PHP files but only in the CSVs not to break the language packs, do you think we should apply it here?

Reply:
I would like to change the PHP file, easier for search.

Reply:
or, can we just not change this string? I think I have a sentimental connection with that string which is there probably since before the pyramids were built, it is really necessary to change it?

Reply:
> or, can we just not change this string? I think I have a sentimental connection with that string which is there probably since before the pyramids were built, it is really necessary to change it?

It's already been changed from Magento to OpenMage... 😁